README.md

```diff
@@ -67,13 +83,82 @@ If you need to access a `psql` shell for inspecting the database, you can use:
 
 #### Nomad
 
-Similarly, you will need to run a local Nomad service instance in a Docker
+Similarly, you will need to run a local
+[Nomad](https://www.nomadproject.io/) service instance in a Docker
 container. You can do so with:
 
 ```bash
 ./run_nomad.sh
 ```
 
+Nomad is an orchestration tool which the Data Refinery uses to run
+Downloader and Processor jobs. Jobs are queued by sending a message to
+the Nomad agent, which will then launch a Docker container which runs
+the job.
+
+
+### Running Locally
+
+Once you have the Postgresql and Nomad services running, you're ready
+to run jobs. There are three kinds of jobs within the Data Refinery.
+
+#### Surveyor Jobs
+
+Surveyor Jobs discover samples to download/process along with
+recording metadata about the samples. A Surveyor Job should queue
+Downloader Jobs to download the data it discovers.
+
+The Surveyor can be run with the `./foreman/run_surveyor.sh` script. The first argument to this script is the type of Surveyor Job to run. The three valid options are:
```

Comment:
Am I crazy or are some of the lines 100 char and some aren't?

Reply:
I missed splitting a couple of these out, so no, not crazy :)